app/src/main/res/layout/activity_navigator.xml

```diff
@@ -159,11 +159,10 @@
         android:layout_marginBottom="@dimen/default_bottom_margin"
         android:clickable="true"
         android:focusable="true"
-        app:backgroundTint="@color/colorPrimary"
+        android:src="@drawable/ic_beacon"
         app:tint="@color/white"
         app:layout_constraintBottom_toTopOf="@+id/navigation_sheet"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:srcCompat="@drawable/ic_beacon" />
```

Comment:
for some reason android:src was able to understand style's tint better, you won't need the compat anyway as you aren't supporting Android 4 anyway.